src/main/java/tc/oc/pgm/rotation/DisabledMapPool.java

```diff
@@ -0,0 +1,25 @@
+package tc.oc.pgm.rotation;
+
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.map.PGMMap;
+
+public class DisabledMapPool extends MapPool {
```

Comment:
Isn't this basically a `RandomMapPool` would that be more descriptive of what this class does?

Reply:
This is not a random map pool, this is an always disabled map pool, it's used when you specify an invalid type of map pool, so it's not a rotation nor a voted pool, so it's an "always disabled map pool fallback", the random is only an ok-ish-enough default if for any reason someone calls pollNextMap, wich should never happen.